.taprc

```diff
@@ -1 +1 @@
-check-coverage: false
+check-coverage: true
```

Comment:
I would just remove the file. Check coverage is `true` by default.

Reply:
removed